application/common/manifest_handlers/tizen_ime_handler.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2014 Samsung Electronics Co., Ltd All Rights Reserved
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_APPLICATION_COMMON_MANIFEST_HANDLERS_TIZEN_IME_HANDLER_H_
+#define XWALK_APPLICATION_COMMON_MANIFEST_HANDLERS_TIZEN_IME_HANDLER_H_
+
+#include <string>
+#include <vector>
+
+#include "base/macros.h"
+#include "xwalk/application/common/application_data.h"
+#include "xwalk/application/common/manifest_handler.h"
+
+namespace xwalk {
+namespace application {
+
+class TizenImeInfo : public ApplicationData::ManifestData {
+ public:
+  TizenImeInfo();
+  virtual ~TizenImeInfo();
+
+  const std::string& Uuid() const {
+    return uuid_;
+  }
+  inline void SetUuid(const std::string& uuid) { uuid_ = uuid; }
```

Comment:
s/SetUuid/set_uid (as far as I remember I had mentioned it before.)
also pls remove the "inline" keyword